### ISSUE TITLE
Chess icons

### ISSUE THREAD

**Issue by eduayme** [twbs/icons#1197]
Icons request:
- [ ] Outline pawn
- [ ] Outline rook
- [ ] Outline knight
- [ ] Outline bishop
- [ ] Outline queen
- [ ] Outline king
- [ ] Filled pawn
- [ ] Filled rook
- [ ] Filled knight
- [ ] Filled bishop
- [ ] Filled queen
- [ ] Filled king
- [ ] Chess board
- [ ] Chess clock

**Comment by GMaggiulli** [comment]
Yes! I would like to see at least the chess pawn icon ♟️ added to the set.
I've found this reference from Material's Icons: [Pawn](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:chess_pawn:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=pawn&icon.size=24&icon.color=%23e3e3e3)
